lib/dbPool.js

```diff
@@ -161,28 +13,30 @@ class DBPool {
    * @constructor
    */
   constructor(database = {
-    url: '*LOCAL'
+    url: '*LOCAL',
```

Comment:
@abmusse Are these trailing commas necessary?

Reply:
Is it part of the linting?

Reply:
@brianmjerome 

Definitely not necessary but did so to because linter complained about [this](https://eslint.org/docs/rules/comma-dangle)

Reply:
@abmusse Yeah I guess I'm just not used to this coding style haha. It's better for future diffs anyway.

Reply:
Agreed it does look really uncessary